src/webview/html/messageAsHtml.js

```diff
@@ -43,6 +43,15 @@ const messageReactionAsHtml = (
       : codeToEmojiMap[reaction.code]
   }&nbsp;${reaction.count}</span>`;
 
+const addReactionButtonAsHtml = (): string =>
+  template` <span class="add-emoji-button" 
+      data-name="plus"
+      data-code="2795"
+      data-type="unicode_type">
```

Comment:
Do these three attributes have any effect? I think probably not, and we should leave them out.

The reason they're there on actual reactions is that they get used when you tap on the reaction, as part of the outbound event telling the RN-side code to add or remove that particular reaction. Search for "data-name" in js.js to see that happening. But we don't want to be trying to manipulate an actual reaction with the emoji `:plus:` :slightly_smiling_face:

Reply:
Done :)